api_tests/lib_sdk/wallets/index.ts

```diff
@@ -65,7 +65,7 @@ export async function pollUntilMinted(
     if (new BigNumber(currentBalance).gte(minimumBalance)) {
         return;
     } else {
-        await sleep(500);
```

Comment:
📣We have a new block every 1 second. Which means requesting the balance every 200ms does not make much difference besides putting more load on your machine.